From 53787c195ff72f06a6a307d5478b56dd000f8376 Mon Sep 17 00:00:00 2001
From: keep <535259334@qq.com>
Date: Tue, 15 May 2018 10:47:00 +0800
Subject: [PATCH] change Admin.java

---
 Admin.java | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Admin.java b/Admin.java
index 1751c01..88229fc 100644
--- a/Admin.java
+++ b/Admin.java
@@ -1 +1,3 @@
-sdfadf
\ No newline at end of file
+sdfadf
+
+sdf
\ No newline at end of file
-- 
2.17.0.windows.1

